data/interactive-defaults.ks

```diff
@@ -5,7 +5,7 @@ firstboot --enable
 
 %anaconda
 # Default password policies
-pwpolicy root --strict --minlen=8 --minquality=50 --nochanges --emptyok
-pwpolicy user --strict --minlen=8 --minquality=50 --nochanges --emptyok
-pwpolicy luks --strict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy root --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy user --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy luks --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
```

Comment:
Sorry, that was perhaps overzealous. I'll send a revision shortly.

One additional question: In order to use --notstrict, do I also have to revert the following commits?
* https://github.com/rhinstaller/anaconda/commit/fe7232768695f477b23a0a64ec6a9fd15e55675c
* https://github.com/rhinstaller/anaconda/commit/42b3d73cd5a2ddc460e5beea8d204d07589023b2 
* https://github.com/rhinstaller/anaconda/commit/c8074a0f5df82aa2791e8411b4fcbe692f8c9643 
* https://github.com/rhinstaller/anaconda/commit/42b3d73cd5a2ddc460e5beea8d204d07589023b2